### PR TITLE
doc(tuto): add example for offsite backup between regions or clouders

### PR DESCRIPTION
**Pull request by samber**
### Your checklist for this pull request

- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Feel free to remove the ovh block if it does not fit your internal policy.

### CODE REVIEW

File: tutorials/migrate-data-rclone/index.mdx [hunk 1]
```diff
@@ -383,4 +383,50 @@ To transfer data to Scaleway Glacier class, add `--s3-storage-class=GLACIER` to
...
+access_key_id = xxxx
+secret_access_key = xxxx
+endpoint = https://s3.nl-ams.scw.cloud
+region = nl-ams
+```
+
+Then sync buckets OVH->SCW and SCW->SCW:
+
+```bash
+# sync from ovh to scaleway (nl-ams)
```
Comment (bene2k1) [inline tutorials/migrate-data-rclone/index.mdx:427]:
```suggestion
# sync from the other Object Storage provider to Scaleway (nl-ams)
```

File: tutorials/migrate-data-rclone/index.mdx [hunk 2]
```diff
@@ -383,4 +383,50 @@ To transfer data to Scaleway Glacier class, add `--s3-storage-class=GLACIER` to
...
+rclone --config rclone.conf sync ovh-gra://my-bucket/my-key-prefix scw-nl-ams://my-bucket/my-key-prefix
+
+# sync from scaleway (fr-par) to scaleway (nl-ams)
```
Comment (bene2k1) [inline tutorials/migrate-data-rclone/index.mdx:430]:
```suggestion
# sync from Scaleway (fr-par) to Scaleway (nl-ams)
```

File: tutorials/migrate-data-rclone/index.mdx [hunk 3]
```diff
@@ -383,4 +383,50 @@ To transfer data to Scaleway Glacier class, add `--s3-storage-class=GLACIER` to
...
+## Offsite backup
+
+The `sync` command from rclone should be prefered to `copy`, because it deletes files from destination.
+
+First, create the `rclone.conf`:
+
+```
+[other-provider]
+type = s3
+provider = OVH
```
Comment (bene2k1) [inline tutorials/migrate-data-rclone/index.mdx:397]:
```suggestion
provider = <PROVIDER_NAME>
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: tutorials/migrate-data-rclone/index.mdx [hunk 4]
```diff
@@ -383,4 +383,50 @@ To transfer data to Scaleway Glacier class, add `--s3-storage-class=GLACIER` to
 rclone copy --progress --s3-storage-class=GLACIER <SOURCE_BACKEND>:<SOURCE_PATH> <DEST_BACKEND>:<DEST_PATH>
 ```
 
-You can verify the storage class of the transferred data by accessing your bucket on the Scaleway console.
\ No newline at end of file
+You can verify the storage class of the transferred data by accessing your bucket on the Scaleway console.
+
+## Offsite backup
+
+The `sync` command from rclone should be prefered to `copy`, because it deletes files from destination.
```
[the remark this answers is not in the document]
Comment (jcirinosclwy) [reply tutorials/migrate-data-rclone/index.mdx:390]: *preferred with two Rs, no?

**Comment by bene2k1** [comment]
Thank you @samber.
Your PR has been approved and merged. 
The content will be added to the documentation with the next update of our website.

Benedikt

**Pull request merged**